cosmos-sdk-proto/Cargo.toml

```
@@ -18,6 +18,8 @@ rust-version = "1.72"
 [dependencies]
 prost = { version = "0.13", default-features = false }
 tendermint-proto = "0.39.1"
+serde = { version = "1.0.203", default-features = false }
```

Probably need `serde`'s `alloc` feature:

```suggestion
serde = { version = "1.0.203", default-features = false, features = ["alloc"] }
```